server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/OldRestoreAction.java

```diff
@@ -105,9 +112,10 @@ private void writeResponse(JsonWriter json, BulkChangeResult result) {
         jsonProfile.prop("languageName", language.getName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)